src/main/resources/org/sagebionetworks/web/client/widget/footer/FooterViewImpl.ui.xml

```diff
@@ -36,7 +36,7 @@
               <!--Acts as a max-width flex column between logo and alpha mode buttons in desktop view. Contains a centered logo in mobile view-->
               <bh:Div addStyleNames="visible-xs margin-auto" width="200px">
                 <g:Image
-                  url="images/logo.svg"
+                  url="/images/logo.svg"
```

Comment:
As well as fix a number of relative paths that are broken after the routing change